v4/export/config.go

```diff
@@ -426,13 +427,14 @@ func (conf *Config) ParseFromFlags(flags *pflag.FlagSet) error {
 		conf.TableFilter = filter.CaseInsensitive(conf.TableFilter)
 	}
 
+	if tidbMemQuotaQuery != 0 {
+		conf.MemQuotaQuery = tidbMemQuotaQuery
+	}
```

Comment:
I think it's better to put these lines after line 382. We don't need `tidbMemQuotaQuery` anymore and can assign this value directly to `conf.MemQuotaQuery`.
What's more, should `conf.MemQuotaQuery` be `conf.TiDBMemQuotaQuery`?